content/legal/release-policy.md

```diff
@@ -71,19 +71,19 @@ contain compiled code.
 
 #### Release signing  {#release-signing}
 
-All supplied packages MUST be cryptographically signed with a detached signature.
-It MUST be signed by either the Release Manager or the automated release
+All supplied packages MUST be cryptographically signed with an ASCII-armored detached signature.
+They MUST be signed by either the Release Manager or the automated release
```

Comment:
Can we mention that a RM signature should use `<username>@apache.org` as its mail address?

Also, how do we view/recomment a multiple addresses signature, like:

```
apache-pekko-grpc-1.0.2-incubating-src-20231229.tgz
gpg: Signature made 六 12/30 00:39:42 2023 CST
gpg:                using RSA key 6BA4DA8B1C88A49428A29C3D0C69C1EF41181E13
gpg: Good signature from "PJ Fanning (GitHub noreply address) <pjfanning@users.noreply.github.com>" [unknown]
gpg:                 aka "PJ Fanning <fanningpj@yahoo.com>" [unknown]
gpg:                 aka "PJ Fanning (http://www.apache.org/) <fanningpj@apache.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 6BA4 DA8B 1C88 A494 28A2  9C3D 0C69 C1EF 4118 1E13
```

@wu-sheng once found that such a signature may not display fully on some signature viewers (that display the first mail address).